src/sentry/hybridcloud/models/apikeyreplica.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+    sane_repr,
+)
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_only_model
+class ApiKeyReplica(Model):
```

Comment:
Will we also need replicas for `OrgAuthToken` and `ApiToken`? Those can also be used in both region and control, with control being the owner.

Reply:
Absolutely.  They'll come in a follow up because I wanted to do each piece incrementally.